src/bin/utils.ts

```diff
@@ -68,7 +69,11 @@ export const getAbstractSqlModelFromFile = (
 			return fileContents;
 		}
 		const configModel =
-			'models' in fileContents ? fileContents.models[0] : fileContents;
+			'models' in fileContents
+				? modelName
+					? fileContents.models.find((m) => m.modelName === modelName)!
```

Comment:
Can we treat the case where we passed a modelName that does not exist and throw a more specific error? Right now it throws with a very cryptic error message like:

```
        if ('abstractSql' in configModel && configModel.abstractSql != null) {
                          ^

TypeError: Cannot use 'in' operator to search for 'abstractSql' in undefined
    at getAbstractSqlModelFromFile (/home/otavio/git/balena-api/node_modules/@balena/pinejs/out/bin/utils.js:87:27)
    at Command.generateTypes (/home/otavio/git/balena-api/node_modules/@balena/pinejs/out/bin/abstract-sql-compiler.js:13:65)
    at async Command.parseAsync (/home/otavio/git/balena-api/node_modules/commander/lib/command.js:936:5)
```